app/views/conditions/dashboard.erb

```diff
@@ -0,0 +1,98 @@
+<div class="container">
+  <h2>Weather Dashboard</h2>
+  <h4>Max Temperature</h4>
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th>Max Temperature (F)</th>
+        <th>Average Number of Rides</th>
+        <th>Highest Number of Rides</th>
+        <th>Lowest Number of Rides</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% (40..80).step(10) do |n| %>
+      <tr>
+        <td><%= "#{n} to #{n + 10}" %></td>
+        <td><%= Condition.average_number_of_rides_from_temp(n) %></td>
+        <td><%= Condition.highest_number_of_rides_from_temp(n) %></td>
+        <td><%= Condition.lowest_number_of_rides_from_temp(n) %></td>
+      </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+  <h4>Precipitation</h4>
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th>Precipitation (inches)</th>
+        <th>Average Number of Rides</th>
+        <th>Highest Number of Rides</th>
+        <th>Lowest Number of Rides</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% (0..(0.5)).step(0.5) do |n| %>
+      <tr>
+        <td><%= "#{n} to #{n + 0.5}" %></td>
+        <td><%= Condition.average_number_of_rides_from_precipitation(n) %></td>
+        <td><%= Condition.highest_number_of_rides_from_precipitation(n) %></td>
+        <td><%= Condition.lowest_number_of_rides_from_precipitation(n) %></td>
+      </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+  <h4>Visibility</h4>
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th>Visibility (miles)</th>
+        <th>Average Number of Rides</th>
+        <th>Highest Number of Rides</th>
+        <th>Lowest Number of Rides</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% (0..10).step(5) do |n| %>
+      <tr>
+        <td><%= "#{n} to #{n + 5}" %></td>
+        <td><%= Condition.average_number_of_rides_from_visibility(n) %></td>
+        <td><%= Condition.highest_number_of_rides_from_visibility(n) %></td>
+        <td><%= Condition.lowest_number_of_rides_from_visibility(n) %></td>
+      </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+  <h4>Wind Speed</h4>
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th>Wind Speed (mph)</th>
+        <th>Average Number of Rides</th>
+        <th>Highest Number of Rides</th>
+        <th>Lowest Number of Rides</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% (0..20).step(5) do |n| %>
+      <tr>
+        <td><%= "#{n} to #{n + 5}" %></td>
+        <td><%= Condition.average_number_of_rides_from_wind_speed(n) %></td>
```

Comment:
again - too much logic in your view - your view should simply be displaying things - not querying your db to get a specific subset of data